### DISCUSSION TITLE
[FEAT] Adding event 'onFirstRender'

### DISCUSSION THREAD

**Discussion by Goliath86** [petyosi/react-virtuoso#975] [Ideas]
What do you think to add a new event named for example 'onFirstRender' that execute a callback after the first rendering of the list? Actually I use the 'itemsRendered' event with a flag to execute a callback only on first render. Is there eventually another method to catch the first render of the list?

**Comment by petyosi** [comment]
Can you explain what you need this for? I should be able to understand the purpose of such event better. An example of what you're doing would be great.
